packages/react-server/README.md

```diff
@@ -156,11 +156,11 @@ We welcome contributions to react-server!  To contribute, follow these steps:
 // TODO: Concepts
 // TODO: routing guide
 // TODO: page guide
-// TODO: generator 
+// TODO: generator
 
-## What is "triton"? Why do I see that in code comments?
+## What is "triton"? Why do I see that in the git history?
 
-Back when we started this project, we came up with a great name -- `triton` -- and we started using it internally. And then it took us forever to get around to making the source code public. And then [Joyent](https://www.joyent.com) released a product called `triton` and stole our thunder, so we had to go with `react-server` instead (a terrible, terrible tragedy, we know). It has always seemed like a preeeety big coincidence to us that they were first to market with a product named `triton`, and they are also one floor above our San Francisco engineering office...
```

Comment:
It does if the goal was to make a silly insinuation that they stole our project name after overhearing us use it in the elevator. (Admittedly, the silly insinuation was poorly executed.)